projects/discussion-ui/src/lib/components/side-pannel/side-pannel.component.ts

```diff
@@ -74,7 +74,8 @@ export class SidePannelComponent implements OnInit, OnDestroy {
     if (event) {
       this.telemetryUtils.logInteract(event, NSDiscussData.IPageName.HOME);
     }
-    this.router.navigate([`${pageName}`], { relativeTo: this.activatedRoute });
+    this.router.navigate([`${CONSTANTS.ROUTES.DISCUSSION}${pageName}`]);
```

Comment:
@sudip-mukherjee @ArtiSingh23 This has to append with routerSlug as well

Refer: [Link line 137](https://github.com/Sunbird-Ed/discussions-UI/blob/41ce6eac368f424b10feadaf950b61ec1e1f64fe/projects/discussion-ui/src/lib/components/my-discussion/my-discussion.component.ts)
let routerSlug = this.configService.getConfig().routerSlug ? this.configService.getConfig().routerSlug : ''
this.router.navigate([`${routerSlug}${CONSTANTS.ROUTES.DISCUSSION}${pageName}`]);

Reply:
no need, here slug the category name/id. 
**" _.get(discussionData, 'topic.slug')}`]); "**